src/messages.rs

```diff
@@ -267,6 +263,9 @@ impl RoutingMessage {
 #[derive(PartialEq, Eq, PartialOrd, Ord, Clone, Debug, RustcEncodable, RustcDecodable)]
 pub struct SignedMessage {
     encoded_body : Vec<u8>,
+    claimant     : Address,
```

Comment:
That is why I corrected myself, that is, it will not be undefined for `Client` case, but will be redundant the same way as it is redundant for `ManagedNode`.